pkg/controller/machineset/machineset_controller.go

```diff
@@ -217,7 +217,10 @@ func (r *ReconcileMachineSet) reconcile(ctx context.Context, machineSet *cluster
 		filteredMachines = append(filteredMachines, machine)
 	}
 
-	syncErr := r.syncReplicas(machineSet, filteredMachines)
+	var syncErr error
+	if machineSet.DeletionTimestamp == nil {
```

Comment:
This is handled on line 139, isn't it?

Reply:
Oh, it is. Apologies for the noise. (Too many branches.)